test/acceptance/shopping-cart.controller.acceptance.ts

```diff
@@ -117,24 +117,26 @@ describe('ShoppingCartController', () => {
     await cartRepo.deleteAll();
   }
 
+  function givenAnItem(item?: Partial<ShoppingCartItem>) {
+    return new ShoppingCartItem(
+      item || {
```

Comment:
Data builders typically use `Object.assign`, allowing users to leverage sensible defaults provided by the data builder and customize only the properties relevant to the test.

```ts
const defaultData = {
 // sensible property values
};
return new ShoppingCartItem(
  Object.assign(defaultData, item)
);
```